test/e2e_node/apparmor_test.go

```diff
@@ -229,10 +229,10 @@ func createPodWithAppArmor(ctx context.Context, f *framework.Framework, profile
 
 func expectSoftRejection(status v1.PodStatus) {
 	args := []interface{}{"PodStatus: %+v", status}
-	framework.ExpectEqual(status.Phase, v1.PodPending, args...)
-	framework.ExpectEqual(status.Reason, "AppArmor", args...)
+	gomega.Expect(status.Phase).To(gomega.Equal(v1.PodPending), args...)
```

Comment:
I wonder whether this is an opportunity to improve stack unwinding: when these assertions fail, pointing towards the `expectSoftRejection` helper might not be useful. Now that we have all features of Gomega available, we could do `gomega.ExpectWithOffset(1, ...)`.

But let's not do that because we don't know whether the test authors want that and because there are multiple assertions in this helper, so knowing which of them failed based on the source code line may be useful.